ftw/linkchecker/command/checking_links.py

```diff
@@ -250,12 +250,33 @@ def add_link_info_to_links(content, link_and_relation_information, obj):
     return link_and_relation_information
 
 
+def extract_relation_uids_in_string(input_string):
+    regex = "resolveuid/\w{32}"
```

Comment:
You could use the regex from plone, which directly extracts the uid:
`resolveuid_re = re.compile('^[./]*resolve[Uu]id/([^/]*)/?(.*)$')`

For referene: https://github.com/plone/plone.outputfilters/blob/master/plone/outputfilters/filters/resolveuid_and_caption.py#L38

Reply:
I'll gonna look at that in issue #30